.kbnconfig

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "0.0.5",
+  "packages": [
+    ".",
+    "packages/*"
```

Comment:
To test this with x-pack, add your x-pack folder here, e.g. `"../x-pack-kibana". When you run `kbn bootstrap` you should get some errors about the versions of our kibana specific eslint packages. Just follow the guidelines there to fix it.